delegates.txt

```diff
@@ -336,7 +336,7 @@ Nikolaus Papaspyrou (NPU)
 Noah Tye (NTE)
 Norbert Lindenberg (NL)
 Oliver Hunt (OH)
-Pablo Gorostiaga Belio (PGB)
+Pablo Gorostiaga Belio (PBO)
```

Comment:
Why override the name selected by the delegate?

Reply:
I asked if it was ok to follow the typical convention -- https://github.com/tc39/notes/pull/267#issuecomment-1682811818

Reply:
Sorry, I missed that and didn't pass it on to the delegate. Did you get in touch with him?

Reply:
I did not, but will.  This PR does not need to be gated on that at all, I just needed to either make the change to `delegates.txt` or to the test code because his TLA PR already merged.